lib/config.js

```diff
@@ -159,27 +158,34 @@ module.exports.fromKubeconfig = fromKubeconfig;
 function mapCertificates(cfgPath, config) {
   const configDir = path.dirname(cfgPath);
 
-  config.clusters.filter(cluster => cluster.cluster['certificate-authority']).forEach(cluster => {
-    cluster.cluster['certificate-authority'] = path.resolve(configDir, cluster.cluster['certificate-authority']);
-  });
+  if (config.clusters) {
+    config.clusters.filter(cluster => cluster.cluster['certificate-authority']).forEach(cluster => {
+      cluster.cluster['certificate-authority'] = path.resolve(configDir, cluster.cluster['certificate-authority']);
+    });
+  }
 
-  config.users.filter(user => user.user['client-certificate']).forEach(user => {
-    user.user['client-certificate'] = path.resolve(configDir, user.user['client-certificate']);
-  });
+  if (config.users) {
+    config.users.filter(user => user.user['client-certificate']).forEach(user => {
+      user.user['client-certificate'] = path.resolve(configDir, user.user['client-certificate']);
+    });
 
-  config.users.filter(user => user.user['client-key']).forEach(user => {
-    user.user['client-key'] = path.resolve(configDir, user.user['client-key']);
-  });
+    config.users.filter(user => user.user['client-key']).forEach(user => {
+      user.user['client-key'] = path.resolve(configDir, user.user['client-key']);
+    });
+  }
 
   return config;
 }
 
 function loadKubeconfig(cfgPath) {
-  cfgPath = cfgPath || defaultConfigPath();
+  const cfgPaths = cfgPath ? [cfgPath] : defaultConfigPath();
```

Comment:
@JacopoDaeli was thinking, should we support passing an array of config paths in here as well? i.e. 

```js
let cfgPaths;
if (!cfgPath) { cfgPaths = defaultConfigPath(); }
else if(Array.isArray(cfgPath)) { cfgPaths = cfgPath; }
else { cfgPaths = [cfgPath]; }
```

Reply:
I was thinking that too. Not sure. In any case, that doesn't hurt, but is it useful?
@silasbw what do you think?